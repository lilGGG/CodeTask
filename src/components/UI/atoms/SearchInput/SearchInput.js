import styled from 'styled-components';

const SearchInp = styled.input`
  font-size: 18px;
  padding: 10px;
  margin: 10px;
  color:gray;
  background: #e5e2e2;
  border: none;
  border-radius: 3px;
  ::placeholder {
    color: chocolate;
  }
`;


function SearchInput({
    inputHandler
}) {
  return (
      <>
        <SearchInp  placeholder="Enter Tv Show Title" onChange={inputHandler} />
      </>
  ) 
}

export default SearchInput;
