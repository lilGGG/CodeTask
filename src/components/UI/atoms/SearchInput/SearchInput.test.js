import React from "react";
import SearchInput from "./SearchInput";
import ReactDom from "react-dom";
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import { create, act } from "react-test-renderer";
import { shallow, mount, render } from "enzyme";

describe('SearchInput test', () => {
    it('renders snapshot', () => {
      const tree = renderer.create(<SearchInput inputHandler={()=> {}}/>).toJSON();
      expect(tree).toMatchSnapshot();
    });

  });
  
