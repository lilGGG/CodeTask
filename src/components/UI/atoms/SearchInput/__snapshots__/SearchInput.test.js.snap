// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchInput test renders snapshot 1`] = `
<input
  className="sc-bczRLJ lcYJBc"
  onChange={[Function]}
  placeholder="Enter Tv Show Title"
/>
`;
